.github/CODEOWNERS

```diff
@@ -62,6 +62,7 @@
 /avm/res/dev-test-lab/lab/ @Azure/avm-res-devtestlab-lab-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/digital-twins/digital-twins-instance/ @Azure/avm-res-digitaltwins-digitaltwinsinstance-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/document-db/database-account/ @Azure/avm-res-documentdb-databaseaccount-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/res/document-db/mongo-cluster/ @Azure/avm-res-documentdb-mongocluster-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
Hey @zedy-wj,
just noticed this file shows errors. The reason is usually that the team in question (in this case `Azure/avm-res-documentdb-mongocluster-module-owners-bicep`) is not setup, or not setup correctly. Could you please check if that team is setup & is placed correct in the hierachy? This is required so that those configured as code-owners can subsequently approve PRs without the need to involve the repository maintainers. The corresponding spec with instructions can be found here: [SNFR20 - Category: Contribution/Support - GitHub Teams Only](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only).

Reply:
@sinedied - Could you please follow the [instructions](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) and create a team named: `avm-res-documentdb-mongocluster-module-owners-bicep` since I'm not authorized (Non-FTE), I can't do that.

Reply:
@zedy-wj All the required teams have been created, it should be good now

Reply:
Hey @AlexanderSehr,
At present, I can indeed check that the team: `avm-res-documentdb-mongocluster-module-owners-bicep` has been setup, but this file still reports an error. Does the owner need any additional steps?

Reply:
@zedy-wj after looking again, it seems it was missing the parent teams addition to get the permissions right, so I added this. Access will have to be granted by the parent teams, then it should work

Reply:
@sinedied, I've just approved these team requests.

Reply:
Just came back for some time off. Looks good & will resolve